internal/apis/kfd/v1alpha2/eks/create/infrastructure.go

```diff
@@ -140,7 +144,7 @@ func (i *Infrastructure) Exec(opts []cluster.OperationPhaseOption) error {
 		}
 
 		for _, opt := range opts {
-			if strings.ToLower(opt.Name) == cluster.OperationPhaseOptionVPNAutoConnect {
+			if strings.ToLower(opt.Name) == cluster.OperationPhaseOptionVPNAutoConnect && opt.Value == true {
```

Comment:
please revert this change as it's fixed in here (PR #126): https://github.com/sighupio/furyctl/pull/126/files#diff-8a6e19e5142516cbc6e52987fe939bf5beaac01b413e7141eca23a9a5c2c83dfR144